File: src/spice2x/api/resources/dart/spiceapi-websocket/src/wrappers/info.dart
part of spiceapi;

Future<Map> infoAVS(Connection con) {
  var req = Request("info", "avs");
  return con.request(req).then((res) {
    return res.getData()[0];
  });
}

Future<Map> infoLauncher(Connection con) {
  var req = Request("info", "launcher");
  return con.request(req).then((res) {
    return res.getData()[0];
  });
}

Future<Map> infoMemory(Connection con) {
  var req = Request("info", "memory");
  return con.request(req).then((res) {
    return res.getData()[0];
  });
}

File: src/spice2x/api/resources/python/spiceapi/info.py
from .connection import Connection
from .request import Request


def info_avs(con: Connection):
    res = con.request(Request("info", "avs"))
    return res.get_data()[0]


def info_launcher(con: Connection):
    res = con.request(Request("info", "launcher"))
    return res.get_data()[0]


def info_memory(con: Connection):
    res = con.request(Request("info", "memory"))
    return res.get_data()[0]
